samples/colab/pytorch_huggingface_whisper.ipynb

```diff
@@ -0,0 +1,374 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": [],
+      "collapsed_sections": [
+        "UUXnh11hA75x",
+        "jbcW5jMLK8gK"
+      ]
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "##### Copyright 2024 The IREE Authors"
+      ],
+      "metadata": {
+        "id": "UUXnh11hA75x"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "#@title Licensed under the Apache License v2.0 with LLVM Exceptions.\n",
```

Comment:
What does the @title do? Are there some ipynb viewers that render this as an actual title? At least in github's preview feature, it just leave the comment as-is.

Reply:
Yeah these are designed for (interactive) viewing in Colab (https://colab.google/). Anyone can open a notebook from the URL and click "Runtime -> Run All" (or Connect -> run individually) to play with the code. There are a few alternatives to Colab for online Jupyter notebooks with different hosting tiers and features, but I'm most familiar with Colab.

Colab renders a single line of markdown following `#@title` above the cell:
![image](https://github.com/openxla/iree/assets/4010439/19a0e1a6-d9d2-4102-9d50-e67a9c70345e)

You can also collapse cells that have titles: 
![image](https://github.com/openxla/iree/assets/4010439/ed1b6e0c-2f1d-47b7-9c90-a7830c738b45)

Could replace those titles embedded into code cells with text cells, but I like the direct link between the title and the code, as well as that collapsing ability. Maybe I'm overusing titles though. Just looked at a few notebooks from JAX and didn't see them in use there.

Reply:
Switched to text cells.

Reply:
Oh, I didn't mean it as a hard suggestion. I just haven't used google collab before. I guess it's nice that the github preview now has the titles. 

I noticed that boxes 1, 2, and 5 still have the inlined markdown, so if you want to be consistent with the change, I would modify those, too.

Reply:
I hid those cells (by default) but kept the titles: 
![image](https://github.com/openxla/iree/assets/4010439/044f4bce-dc1b-4ab6-aa34-8f50878b6ecb)

e.g. copyright hidden vs fully expanded:
![image](https://github.com/openxla/iree/assets/4010439/5fc61f0e-19b5-4608-97bb-acd379201c06)
![image](https://github.com/openxla/iree/assets/4010439/634384ad-3f43-45b6-9481-f353a95a877c)

not really expecting people to focus on those sections, so I want the collapsing behavior